### PR TITLE
feat: Add initial grpc call example

### PR DESCRIPTION
**Pull request by nand4011**
Add generated grpc code into a new directory.

Add grpctest.exs in the examples directory that makes a set and get call.

### CODE REVIEW

File: src/generated/cacheping.pb.ex [hunk 1]
```diff
@@ -0,0 +1,85 @@
+defmodule CacheClient.PingRequest do
```
Comment (cprice404) [inline src/generated/cacheping.pb.ex:1]: can you please add a README in the `generated` dir that explains how you got these?

File: src/mix.exs [hunk 2]
```diff
@@ -18,10 +19,15 @@ defmodule Momento.MixProject do
...
+  defp elixirc_paths(_), do: ["lib", "generated"]
+
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:dialyxir, "~> 1.3", only: [:dev], runtime: false}
+      {:dialyxir, "~> 1.3", only: [:dev], runtime: false},
+      {:grpc, "~> 0.5.0"},
+      {:protobuf, "~> 0.12.0"},
+      {:google_protos, "~> 0.3"}
```
Comment (cprice404) [inline src/mix.exs:30]: so all of these are runtime deps, yeah? just making sure none of them are only required for protogen

File: src/generated/cacheping.pb.ex (hunk 1 above)
Comment (nand4011) [reply src/generated/cacheping.pb.ex:1]: Done.

File: src/mix.exs (hunk 2 above)
Comment (nand4011) [reply src/mix.exs]: I removed protobuf and google_protos and the example still runs after a mix deps.get. We can add them back in if they are required in the future.

Comment (nand4011) [reply src/mix.exs]: Nevermind. I must have some lingering state locally. They are both required.

File: src/generated/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Generated gRPC Files
+
+These files were generated from [Momento Client Protos](https://github.com/momentohq/client-protos). They will be moved 
+into that project in the future.
+
+They were made using the protoc tool and [protobuf-elixir](https://github.com/elixir-protobuf/protobuf).
```
Comment (cprice404) [inline src/generated/README.md:6]: Will it be obvious for someone to figure out the necessary commands? Or can you just include them here?

Comment (nand4011) [reply src/generated/README.md:6]: I'll add them in.

**Pull request merged**